app/lib/actions/foreman/puppetclass/import.rb

```diff
@@ -0,0 +1,32 @@
+module Actions
+  module Foreman
+    module Puppetclass
+      class Import < Actions::EntryAction
+        def resource_locks
+          :import_puppetclasses
+        end
+
+        def plan(params)
+          plan_self(params)
```

Comment:
This is the dafult implementation of the plan method in the action - no need to redefine unless other intention